src/backend/OpenCL/CHIPBackendOpenCL.hh

```diff
@@ -151,11 +163,16 @@ public:
 };
 
 class CHIPContextOpenCL : public chipstar::Context {
-public:
-  bool allDevicesSupportFineGrainSVM();
+private:
+  cl::Context ClContext;
+  bool SupportsIntelUSM;
```

Comment:
Is 'Supports' a wrong word here if you want to store which API the allocation was done with; IntelUSMAllocated or similar might be more descriptive?

Reply:
fixed